internal/exec/stages/disks/luks.go

```diff
@@ -294,36 +300,53 @@ func (s *stage) createLuks(config types.Config) error {
 			// pass the device to clevis. We have to loop each device as
 			// the devices could be on different NICs that haven't come
 			// up yet.
+
+			// List of tang servers proven to be reachable.
```

Comment:
Not a list anymore.  I think the variable name is enough and we can just drop this.